pkgs/desktops/gnome-3/apps/gedit/block-update-desktop.patch

```diff
@@ -0,0 +1,15 @@
+--- a/build-aux/meson/post_install.py
++++ b/build-aux/meson/post_install.py
+@@ -17,9 +17,9 @@ if 'DESTDIR' not in os.environ:
+     subprocess.call(['gtk-update-icon-cache', '-qtf',
+                      os.path.join(datadir, 'icons', 'hicolor')])
+ 
+-    print('Updating desktop database...')
+-    subprocess.call(['update-desktop-database', '-q',
+-                     os.path.join(datadir, 'applications')])
++    # print('Updating desktop database...')
++    # subprocess.call(['update-desktop-database', '-q',
```

Comment:
Why do this when we have https://github.com/NixOS/nixpkgs/commit/69be72a64aa7ee9a9ce9c50429bbac8a64ca250d?

Reply:
Hmm, right the commit message is misleading, the problem was most likely not having access to `update-desktop-database`. Might be better to add `desktop-file-utils` to nativeBuildInputs I guess.

Reply:
Yep, and then the setup hook will remove this file so we don't have collisions.